selfdrive/sensord/tests/test_sensord.py

```diff
@@ -73,6 +87,105 @@ def test_sensors_present(self):
 
     self.assertIn(seen, SENSOR_CONFIGURATIONS)
 
+  @with_processes(['sensord'])
+  def test_lsm6ds3_100Hz(self):
+    # verify samples arrive in a 100Hz rate
+    events = read_sensor_events(3) # 3sec (about 300 measurements)
+
+    data_points = set()
+    for event in events:
+      for measurement in event.sensorEvents:
+        # Filter out unset events
+        if measurement.version == 0:
+          continue
```

Comment:
yes, you are right, we don't send empty packages, was a leftover from old code, removed

Reply:
correction, this still happens on devices with the bmx sensor, cause we preallocate the message buffer,
and for the bmx magn sensor it is not sure that it will fill it